src/uct/base/uct_component.h

```diff
@@ -41,6 +41,8 @@ struct uct_md_component {
     ucs_config_global_list_entry_t md_config;  /**< MD configuration entry */
     ucs_list_link_t        tl_list;            /**< List of uct_md_registered_tl_t */
     ucs_list_link_t        list;
+    uint64_t               cap_flags;          /**< Capability flags as defined
```

Comment:
let's just call it 'flags' instead of cap flags in all places (including API)
we don't know which kind of flags we would want to add, not all may be capabilities